testing/README.md

```diff
@@ -0,0 +1,75 @@
+### E2E tests
+
+#### Dependencies
+
+[oblt-cli](https://studious-disco-k66oojq.pages.github.io)
+
+### Try it locally
+
+0. Setup vault
+Export the environment variable `VAULT_TOKEN` or have the `~/.vault-token` file created.
+For further details about how to configure Vault check [the Vault documentation](https://github.com/elastic/infra/tree/master/docs/vault)
+
+1. Install and configure [oblt-cli](https://studious-disco-k66oojq.pages.github.io)
+
+2. Create a cluster for e2e tests:
+
+In the `testing` directory:
+
+```
+GITHUB_TOKEN=... SLACK_CHANNEL=... make create-cluster
```

Comment:
Is the `SLACK_CHANNEL` parameter here required or optional? Seems unnecessary if you are creating a cluster for personal testing.

Reply:
It's required for the `oblt-cli`. We can probably omit it in the future